### DISCUSSION TITLE
Change @OutputFileDirectory after Jobs were created

### DISCUSSION THREAD

**Discussion by Hooch76** [olahallengren/sql-server-maintenance-solution#473] [Questions & Answers]
Hi,

is there a way to change the value for @OutputFileDirectory after the SQL Agent Jobs were created? If so: how? Where is that information stored? (IOW: I want to change the directory where the logfiles are written to)

Thanks, 
Tom

**Comment by olahallengren** [comment]
You can change it in the job properties.
https://docs.microsoft.com/en-us/sql/ssms/agent/job-step-properties-new-job-step-advanced-page
